src/renderers/webxr/WebXRController.js

```diff
@@ -269,6 +232,51 @@ class WebXRController {
 
 			}
 
+			if ( targetRay !== null ) {
```

Comment:
Why is it necessary to move the entire if clause further down in `update()`?

Reply:
To be able to access gripPose without having to call `getPose(inputSource.gripSpace)` twice or to disrupt the current flow of the clause where it gets handled

Reply:
There's an additional code added https://github.com/mrdoob/three.js/pull/24056/files#diff-5fa11b4a2b3fc499aa3a6c03b3e7d4caf9724ba2636fea4d577d93ceb2184e65R239-R244

Reply:
@Mugen87 could you look at my explanation please? 🙂

Reply:
Sorry, I'm not super happy with applying these changes to `WebXRController` knowing it's actually a device issue. Wouldn't it be better to report the issue to OpenXR instead? 

Besides, I still don't understand why the mentioned code block needs to be moved. AFAICS, `getPose(inputSource.gripSpace)` would be called twice in any event when no hand tracking is active.

Reply:
Do you the place where exactly I can report it? I tried asking in webxr discord, but never got an answer.

I totally understand reluctance to insert a workaround for specific device in lib code, but it seems to work ok and doesn't seem to break anything, at least from a glance, otherwise I wouldn't even bother. That being said, there's no pressure from my side.

> AFAICS, getPose(inputSource.gripSpace) would be called twice in any event when no hand tracking is active.
Sorry, but I don't see how. There's 2 calls to `getPose` in `update`, first is for target ray `inputPose = frame.getPose( inputSource.targetRaySpace, referenceSpace );`, second is for grip: `gripPose = frame.getPose( inputSource.gripSpace, referenceSpace );`. I just swapped places them so the target ray can use gripPose instead.

I don't really understand how expensive this call can be, so maybe calling it twice is an option (especially that it will only be called if targetPose is null). If you think this is the way please let me know, I'll change the code

Reply:
> Sorry, but I don't see how. There's 2 calls to getPose in update, first is for target ray inputPose = frame.getPose( inputSource.targetRaySpace, referenceSpace );, second is for grip: gripPose = frame.getPose( inputSource.gripSpace, referenceSpace );. I just swapped places them so the target ray can use gripPose instead.

After reading this and and closer look to the code I finally understand the issue 😅 . Thanks for your patience^^.

Moving the code block is indeed right.

Reply:
> Do you the place where exactly I can report it? I tried asking in webxr discord, but never got an answer.

Have you tried: https://forum.vive.com/forum/92-vive-openxr/

Reply:
@toji If we detect device specific issues/bugs when using WebXR, should we directly report this to the vendor? Or is that something we can report to the Chromium bug tracker (e.g. `Blink>WebXR>VR `)?

Reply:
If it's an error that's only showing up on a specific device or with a certain runtime then most likely the "correct" people to fix it is the vendor, but you can file an issue with Chromium regardless to ensure that we haven't over-specialized our OpenXR implementation to known runtimes rather than the spec.